packages/email/src/templates/new-commission-alert-partner.tsx

```diff
@@ -70,27 +67,61 @@ export default function NewSaleAlertPartner({
             </Section>
 
             <Heading className="mx-0 my-7 p-0 text-lg font-medium text-black">
-              You just made a {earningsInDollars} referral commission!
+              You just earned {earningsInDollars} in commissions!
             </Heading>
 
-            <Text className="text-sm leading-6 text-neutral-600">
-              Congratulations! Someone made a{" "}
-              <strong className="text-black">{saleAmountInDollars}</strong>{" "}
-              purchase on <strong className="text-black">{program.name}</strong>{" "}
-              using your referral link (
-              <a
-                href={partner.referralLink}
-                className="text-semibold font-medium text-black underline"
-              >
-                {getPrettyUrl(partner.referralLink)}
-              </a>
-              ).
-            </Text>
+            {!["custom", "click"].includes(commission.type) && (
+              <Text className="text-sm leading-6 text-neutral-600">
+                Congratulations! Someone{" "}
+                {commission.type === "lead" ? (
+                  "signed up"
+                ) : (
```

Comment:
Should we come up with more generic wording than "signed up" in case the program leads are triggered by something else?